### PR TITLE
chore: update configs

### PR DESCRIPTION
**Pull request by ghostdevv**
## Description

**Don't squash merge**

## Related Issue(s)

Closes [MKWEB-42](https://linear.app/gitpod/issue/MKWEB-42/prettier-config-changes) 

Fixes # 

## Release QA Checklist ✔️

<details>
  <summary>QA Checklist </summary>

## [Priority: High] Functional Testing

- [ ] Test all links to make sure they are working correctly
- [ ] Verify that all forms (sales, support, docs feedback)  are functioning properly and data is being captured correctly
- [ ] Check all buttons and calls-to-action to ensure they are leading to the correct pages
- [ ] Ensure that the website is mobile responsive and functions properly on different devices
- [ ] Ensure that page is optimal for Light/Dark Mode. ⇒ Text/Media Changes

## [Priority: High] Content Testing

- [ ] Check all website copies for accuracy and spelling/grammar errors
- [ ] Verify that all images and videos are displaying properly
- [ ] Ensure that all downloadable files are working correctly
- [ ] Check that all external links are functioning properly

## [Priority: Medium] Performance Testing

- [ ] Try to use optimal media assets. ([Decision Record - How To](https://github.com/gitpod-io/website/blob/main/docs/adr/0003-optimal-assets-format.md))
- [ ] Test website load times to ensure that the website is loading quickly
- [ ] Verify that the website can handle high traffic volumes without crashing
- [ ] Check that all website functionality is working properly under heavy load
- [ ] Perform cross-browser testing to ensure that the website works properly on all major browsers (Goolge Chrome, Firefox, Edge)

## [Priority: Medium] Accessibility Testing

- [ ] Ensure that the website meets basic accessibility guidelines.
- [ ] Test that users with disabilities can easily navigate and use the website
- [ ] Verify that all website content can be accessed using assistive technologies

By including these key items in your website release QA checklist, you can ensure that your website is fully functional, user-friendly, and meets your business requirements.

## Security Testing

> **Note**:💡 If we are changing something related to cookies or data storing or analytics side.

- [ ] Verify that the website is HTTPS secure and that there are no security vulnerabilities
- [ ] Ensure that all sensitive data is being stored and transmitted securely

  </details>

<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/3794"></a>

### CODE REVIEW

**Comment by ghostdevv** [comment]
Changes:
- Added new prettier config & vscode format on save settings
- Formatted files and added to git blame ignore
- Added a new lint script that runs prettier check and svelte-check
- Added a new github action for linting

On the note of the github action, IMO we should remove the husky hooks as we don't need them and they are slow in my experience - WDYT @Siddhant-K-code

**Review by Siddhant-K-code — COMMENTED** [review]
please fix Code conflicts due to recent merges 🫠

**Comment by Siddhant-K-code** [comment]
> Don't squash merge

🤔 why?

**Comment by ghostdevv** [comment]
We can squash merge I will just have to update the commit hash after

**Comment by Siddhant-K-code** [comment]
ah! It's super hard to merge this. Code conflicts again 🫠

**Comment by ghostdevv** [comment]
I'll resolve the conflicts on Wednesdsay and we can merge before we start the migration :pray:

**Comment by Siddhant-K-code** [comment]
@ghostdevv Can you please test the Preview, if all things are working fine? 

I've tested feedback, whitepaper, and sales form. Can you also do a quick check mentioned in PR Template 🙏🏼

File: .prettierignore [hunk 1]
```diff
@@ -1 +1,12 @@
...
+/node_modules/
+/src/functions/render
+build/
+.idea
+stars-cache.json
+.env
+package.json
+package-lock.json
+static/global.css
+static/styles.css
```
Comment (Siddhant-K-code) [inline .prettierignore:12]: add `_redirects` file also.

```suggestion
static/styles.css
_redirects

```

**Review by Siddhant-K-code — COMMENTED** [review]
please ignore formatting SVG files (& all svelte files under `src/lib/components/svgs`) also.

image

**Comment by ghostdevv** [comment]
Where are you seeing that svg error @Siddhant-K-code ?

**Comment by Siddhant-K-code** [comment]
> Where are you seeing that svg error @Siddhant-K-code ?

Open netlify preview, you will be able to see this error in browser console

**Comment by Siddhant-K-code** [comment]
It seems to have an error on the `/contact/sales` page (it already exists in the past). We are good to go.

**Review by Siddhant-K-code — APPROVED** [review]

**Pull request merged**